build/version.props

```diff
@@ -1,6 +1,14 @@
 <Project>
   <PropertyGroup>
-    <!-- version of all grpc-dotnet packages -->
+
+    <!-- package version of grpc-dotnet -->
```

Comment:
Can we do something like https://github.com/aspnet/AspNetCore/blob/master/eng/Versions.props#L9-L13

Let's separate out major, minor, patch and pre-release label and use those to construct the GrpcDotnet*Version properties.

Reply:
Maybe, but something to think about in the future.